bolt/lib/Profile/YAMLProfileReader.cpp

```diff
@@ -420,6 +456,15 @@ Error YAMLProfileReader::readProfile(BinaryContext &BC) {
       errs() << "BOLT-WARNING: profile ignored for function " << YamlBF.Name
              << '\n';
 
+  if (opts::Verbosity >= 2) {
```

Comment:
Since the output is not too verbose, we can start printing at `opts::Verbosity >= 1`.